README.md

```diff
@@ -96,3 +96,5 @@ SimpleCov.start
 ```
```

Comment:
I can see you have been frequently committing and your file structure seems to be correct as well as the naming of your files.